projects/components/CHANGELOG.MD

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
+- BREAKING: internalFactory has been removed from WidgetObject
```

Comment:
Should we mention that this changes the interface for all of our exported widgets?